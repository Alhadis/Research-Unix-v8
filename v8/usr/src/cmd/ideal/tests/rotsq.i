rotsq
.nf
.IS
box square{var ne, nw, se, sw;
ne = nw + size;
se = sw + size;
ne = se + (0,1)*size;
conn ne to se;
conn se to sw;
conn sw to nw;
conn nw to ne;
}

box main{var anchor;
anchor = (1.5,-3);
put square{var size;nw=anchor;size = (1,1);conn (nw + ne)/2 to 0.5*(sw + se);};
}
.IE
.IS
...forget square
.IE
