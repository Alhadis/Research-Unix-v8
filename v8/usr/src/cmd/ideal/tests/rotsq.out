rotsq
.nf
.IS
...maxx 3.5
...maxy -2
...minx 1.5
...miny -4
...line 2.5 -2 1.5 -3
...line 1.5 -3 2.5 -4
...line 2.5 -4 3.5 -3
...line 3.5 -3 2.5 -2
...line 3 -3.5 2 -2.5
.IE
.IS
.IE
